python/grapher.go

```diff
@@ -274,3 +309,19 @@ type RawRef struct {
 	End       uint32
 	ToBuiltin bool
 }
+
+func installPipRequirements(pipBin string, requirementFiles []string) {
+	for _, requirementFile := range requirementFiles {
+		err := runCmdStderr(exec.Command(pipBin, "install", "-r", requirementFile))
+		if err != nil {
+			log.Printf("Error installing dependencies in %s. Trying piecemeal install", requirementFile)
+			if b, err := ioutil.ReadFile(requirementFile); err == nil {
+				for _, req := range strings.Split(string(b), "\n") {
+					runCmdLogError(exec.Command(pipBin, "install", req))
```

Comment:
- [ ] It makes sense for installPipRequirements to fail if this command fails. Then the caller can decide whether to ignore the error.